include/arc_utilities/arc_helpers.hpp

```diff
@@ -586,6 +588,63 @@ namespace arc_helpers
         }
     }
 
+    template<typename ColorType>
+    inline ColorType Multiply(const ColorType& color, const float factor)
+    {
+        ColorType ret;
+        ret.r = TrimColorValue(factor * color.r);
+        ret.g = TrimColorValue(factor * color.g);
+        ret.b = TrimColorValue(factor * color.b);
+        ret.a = TrimColorValue(factor * color.a);
```

Comment:
Multiplying the alpha channel is unexpected. Only multiply the RGB values.

Reply:
Will ponder, I think I agree.